Classes/View/PdfView.php

```diff
@@ -148,7 +162,12 @@ protected function getPdfObject() {
         /* @var $pdf \mPDF */
         $pdf = $this->objectManager->get('mPDF', '', $pageFormat . '-' . $pageOrientation);
         $pdf->SetMargins($leftMargin, $rightMargin, $topMargin);
-        $pdf->CSSselectMedia = $styleSheet;
+
+        if ($styleSheet == 'print' || $styleSheet == 'screen') {
```

Comment:
There's still a default value for `$styleSheet` in line 160. Should that be changed, too?